### PR TITLE
feat: add text to middle, and fix android bugs

### PR DESCRIPTION
**Pull request by danielgek**
Sean i included the other plugin fixies in this one.

This add text to the center of the circle

This also fixies 2 issues with the plugin on android.

The first one is to fixies the bar with(now its the same as rim).
The second is to hide the text in the center of the plugin(set textSize to 0).

Also added some import fixies because of ci

### CODE REVIEW

**Comment by danielgek** [comment]
@sean-perkins can you take a look at it ?

File: src/animated-circle.ios.ts [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
     initNativeView() {
+
+    }
+
+    onLayout(left, top, right, bottom) {
+        super.onLayout(left, top, right, bottom);
+        console.log(`XML: dimensions ${this.effectiveWidth}x${this.effectiveHeight}`);
+        let dpWidth = utils.layout.toDeviceIndependentPixels(this.effectiveWidth);
+        let dpHeight = utils.layout.toDeviceIndependentPixels(this.effectiveHeight);
+        console.log(`XML: dimensions ${dpWidth}x${dpHeight}`); // correct! exactly what user specified
```
Comment (sean-perkins) [inline src/animated-circle.ios.ts]: Let's remove logs so they don't get distributed in the bundles.

File: src/animated-circle.ios.ts [hunk 2]
```diff
@@ -1,25 +1,51 @@
...
+
+
+        this._label.frame = CGRectMake(0, 0, dpWidth, dpHeight);
+        // this._label.text = 'asd ';
```
Comment (sean-perkins) [inline src/animated-circle.ios.ts]: Remove this line.

File: src/animated-circle.ios.ts [hunk 3]
```diff
@@ -1,25 +1,51 @@
...
         this._ios = DRCircularProgressView.new();
         this._ios.alternativeColor = new Color('#FF5722').ios;
         this._ios.progressColor = new Color('#3D8FF4').ios;
+        this._ios.backgroundColor = new Color('#348F74').ios;
         this._ios.thickness = 5;
+
+
+        this._label = UILabel.alloc().initWithFrame(CGRectMake(0, 0, 0, 0));
+        this._label.textAlignment = NSTextAlignment.Center;
+        this._label.textColor = new Color('#3D8FF4').ios;
```
Comment (sean-perkins) [inline src/animated-circle.ios.ts:26]: Can we make the text color configurable for the end user?

File: src/animated-circle.android.ts [hunk 4]
```diff
@@ -33,6 +37,14 @@ export class AnimatedCircle extends Common {
     }
 
     initNativeView() {
+        this.android.setAutoTextSize(false);
+        this.android.setTextMode(at.grabner.circleprogress.TextMode.TEXT);
+        this.android.setTextScale(1.1);
+        this.android.setTextSize(300);
+        this.android.setUnitVisible(false);
+        // this.android.setUnit('bam2');
```
Comment (sean-perkins) [inline src/animated-circle.android.ts]: Clean-up unused comments.

File: src/animated-circle.ios.ts (hunk 3 above)
Comment (danielgek) [reply src/animated-circle.ios.ts:26]: it is configurable, check [animated-circle.ios.ts#L120](https://github.com/danielgek/nativescript-animated-circle/blob/feat-text/src/animated-circle.ios.ts#L120)

**Comment by danielgek** [comment]
addressed comments 👍

**Pull request merged**